cmd/manager/main.go

```diff
@@ -286,9 +286,11 @@ func main() {
 	if err := viper.ReadInConfig(); err != nil { // Handle errors reading the config file
 		setupLog.Info("unable to read in config, errors ignored")
 	}
+	intctrlutil.ReloadRegistryConfig()
```

Comment:
Apply these changes to the main function of dataprotection (it runs in a separate process).

Reply:
done